pkg/source/imgpkg.go

```diff
@@ -19,19 +19,36 @@ package source
 import (
 	"context"
 	"fmt"
+	"net/http"
 	"path"
+	"time"
 
 	"github.com/cppforlife/go-cli-ui/ui"
 	regname "github.com/google/go-containerregistry/pkg/name"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
-	"github.com/k14s/imgpkg/pkg/imgpkg/plainimage"
-	"github.com/k14s/imgpkg/pkg/imgpkg/registry"
+	"github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/plainimage"
+	"github.com/vmware-tanzu/carvel-imgpkg/pkg/imgpkg/registry"
 )
 
 func ImgpkgPush(ctx context.Context, dir string, excludedFiles []string, image string) (string, error) {
-	options := RetrieveGgcrRemoteOptions(ctx)
-	// TODO support more registry options
-	reg, err := registry.NewRegistry(registry.Opts{VerifyCerts: true}, options...)
+
+	options := RetrieveRegistryOptions(ctx)
```

Comment:
**question** these default opts are so similar to the opts defined in the test case why do we need custom options for tests?